client/components/AssignmentItem/index.jsx

```diff
@@ -87,21 +98,26 @@ export const AssignmentItem = ({
                     <PriorityLabel
                         item={assignment}
                         priorities={priorities}
-                        tooltipFlow="down"
+                        tooltipFlow="right"
                     />
 
                     <StateLabel item={assignment.assigned_to} />
                     {hasContent && <Label text="Content" isHollow={true} iconType="darkBlue2" /> }
                     <span className="ListItem__headline">
-                        <i className="icon-time sd-list-item__inline-icon"/>
+                        <span data-sd-tooltip={`Due: ${scheduledDate()}`} data-flow="right">
+                            <i className="icon-time sd-list-item__inline-icon" />
+                        </span>
                         <InternalNoteLabel
                             item={assignment}
                             prefix="planning."
                             marginRight={false}
                         />
-                        {get(assignment, 'planning.scheduled') ? (
-                            <AbsoluteDate date={get(assignment, 'planning.scheduled').toString()} />
-                        ) : (<time><span>{gettext('\'not scheduled yet\'')}</span></time>)}
+                        <span title={gettext('Due Date')}>
```

Comment:
We are showing the exact same date in the tooltip above. Is it expected to show the same information again next to each other ?